### ISSUE TITLE
Multiple File Upload error

### ISSUE THREAD

**Issue by ricardogalvez** [avocode/avocode#592]
IMPORTANT: Please try to describe how did it happen.
I was uploading numerours(5) .psb files into a project in one go. 4 of the files seem to have been accepted and loaded the 5th is hanging at 64% However after the error prompt (and subsequently this message dialog) were fired, the file upload finished... So I'm not sure if its an issue, its the first time this has happened w/ the beta release I'm working in...
-----
App v.2.5.0
Source UID: 126398
```
Error: incorrect header check
  at Zlib._handle.onerror (zlib.js:345:17)

```

**Comment by vojtatranta** [comment]
We have seen this error before, we have it on gitla.
More info can be found here #447 .
Are you sure that it is the problem of uploading multiple PSDs or is it just one, that is corrupted?

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
